pkg/storaged/logs-panel.jsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React from "react";
+
+import { journalctl, renderer } from "journal";
+
+const _ = cockpit.gettext;
+
+// XXX - turn this into something reusable
```

Comment:
... for sharing with systemd/logs? Yes, please :-) But doesn't that require porting systemd/logs.js to React first? But this PR does introduce quite some conceptual duplication with pkg/lib/journal.js, but that one assumes mustache. Untangling this shouldn't happen in this PR for sure, I'm just curious if you have plans in that direction?

Reply:
No concrete plans.  The comment can probably be removed, it was from a time when the code wasn't split into `LogsPanel` and `StorageLogsPanel`.  `LogsPanel` is perfectly reusable, but it should move into `lib/` for that.  I'll do that, even if it has only one user right now.

Reply:
Done.